poster/styles/styles.scss

```diff
@@ -11,480 +7,451 @@ sans-serif;
 $thick-display:'Raleway',
 sans-serif;
 
-/*body text font size and weight*/
-$font-size: 1.7vw;
-$font-weight: 200;
-
-
-/*left and right margins for body (set in span tag)*/
-.body-lineone {
-    margin-top: 6vw;
-}
-
-.body-linetwo {
-    margin-left: 4vw;
-}
-
-.body-linethree {
-    margin-left: 8vw;
-}
-
-.body-linefour {
-    margin-top: 2vw;
-    margin-left: 30vw;
-}
-
-.body-linefive {
-    margin-top: -2vw;
-    margin-left: 16vw;
-}
-
-.body-linesix {
-    margin-left: 4vw;
-}
-
-.body-lineeight {
-    margin-left: 4vw;
-}
-
-.body-lineten {
-    margin-left: 12vw;
-}
-
-.body-lineeleven {
-    margin-left: 8vw;
-}
-
-.body-linetwelve {
-    margin-left: 12vw;
-    margin-bottom: -2vw;
-}
-
-.body-linethirteen {
-    margin-top: -1.5vw;
-    margin-left: 12vw;
-}
-
-.body-linefourteen {
-    margin-left: 8vw;
-}
-
-.body-linefifteen {
-    margin-left: 4vw;
-}
-
-.body-linesixteen {
-    margin-left: 8vw;
-}
-
-.body-lineeighteen {
-    margin-left: 4vw;
-}
-
-.body-linenineteen {
-    margin-left: 6vw;
-}
-
-.body-linetwenty {
-    margin-left: 12vw;
-}
-
-.body-linetwentyone {
-    margin-left: 16vw;
-}
-
-.body-linetwentytwo {
-    margin-top: 10vw;
-    margin-left: 20vw;
-}
-
-.body-linetwentythree {
-    margin-top: 0;
-    margin-left: 16vw;
-}
-
-.body-linetwentyfour {
-    margin-top: -1vw;
-    margin-left: 14vw;
-    margin-bottom: -1.5vw;
-}
-
-.body-linetwentyfive {
-    margin-left: 7vw;
-}
-
-.body-linetwentysix {
-    margin-left: 10vw;
-}
-
-.body-linetwentyseven {
-    margin-left: 8vw;
-}
-
-.body-linetwentyeight {
-    margin-left: 6vw;
-}
-
-.body-linetwentynine {
-    margin-left: 4vw;
-}
-
-.body-linethirty {
-    margin-left: 16vw;
-    margin-bottom: -1.5vw;
-}
-
-.body-linethirtyone {
-    margin-left: 8vw;
+body {
+    background: #202221;
 }
 
-.body-linethirtytwo {
-    margin-left: 4vw;
+/*header in top half*/
```

Comment:
Note that most of the styles themselves haven't changed, I just rearranged the order. You can step through my commits (I can show you how to do this in class) to better understand what I changed.